src/libs/antares/timeelapsed/i_writer.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "info.h"
+
+namespace TimeElapsed
+{
+class IWriter
+{
+public:
+    virtual ~IWriter() = default;
+    virtual void flush() = 0;
```

Comment:
As said earlier, should contain file path and file content as private data members